bundles/ranvier-vendors/behaviors/npc/vendor.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+
+module.exports = (srcPath) => {
+  const Logger = require(srcPath + 'Logger');
+  const B      = require(srcPath + 'Broadcast');
+  return  {
+    listeners: {
+      playerEnter: state => function (config, player) {
+        if (config.enterMessage) {
+          B.sayAt(player, config.enterMessage);
```

Comment:
Yeah I like that, that way it's more like the NPC is talking directly to the player, instead of spamming the room